packages/calcite-components/src/components/stepper-item/stepper-item.e2e.ts

```diff
@@ -36,4 +38,39 @@ describe("calcite-stepper-item", () => {
     await stepperItem.click();
     expect(stepperItemSelect).toHaveReceivedEventTimes(1);
   });
+
+  const stepperItemHTML = html`
+    <calcite-stepper-item heading="Confirm and complete" icon numbered description="Some subtext">
+      <calcite-notice open width="full">
+        <div slot="message">Step 1 Content Goes Here</div>
+      </calcite-notice>
+    </calcite-stepper-item>
+  `;
+  describe("theme", () => {
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-stepper-item-description-text-color": {
+          shadowSelector: `.${CSS.stepperItemDescription}`,
+          targetProp: "color",
+        },
+        "--calcite-stepper-item-heading-text-color": {
+          shadowSelector: `.${CSS.stepperItemHeading}`,
+          targetProp: "color",
+        },
+        "--calcite-stepper-item-icon-color": {
+          shadowSelector: `calcite-icon`,
+          targetProp: "color",
```

Comment:
This should be targeting the mapped subcomponent token (`--calcite-icon-color`).